src/main/java/org/sadun/util/polling/DirectoryPoller.java

```diff
@@ -533,7 +535,9 @@ private void runCycle() {
 
 				File fls[] = dir.listFiles(filter);
 				if (fls == null) {
-					System.err.println((new StringBuilder()).append("Warning: directory ").append(dir).append(" does not exist").toString());
+					String warningMsg = String.format("Warning: directory %s does not exist", dir.toString());
+					logger.warn(warningMsg);
+					System.err.println(warningMsg);

```

Comment:
Please remove the System.err message. Either just use the logger.war or use logger.error